clang/test/dpct/helper_files_ref/include/device.hpp

```diff
@@ -640,6 +640,44 @@ static inline unsigned int get_device_id(const sycl::device &dev){
   return dev_mgr::instance().get_device_id(dev);
 }
 
+inline void
```

Comment:
Add comment for this helper function, as it is public API.

Reply:
fixed